lib/lotus/cli.rb

```diff
@@ -79,6 +79,19 @@ def new(name = nil)
       end
     end
 
+    desc 'generate', 'generates a new action'
+    method_option :path,                desc: 'applications path',        type: :string, default: 'apps'
```

Comment:
@jodosha what do you think about about removing this option? IMHO, users tend to likely to generate new action at LOTUS_ROOT path